src/static-pages/components/ArticleTypes.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Interwave from 'interweave';
+import { useTranslation } from 'react-i18next';
+import { Paragraph } from '../../ui/atoms';
+
+const ArticleTypes = (): JSX.Element => {
+    const { t } = useTranslation('author-guide');
+
+    return (
+        <div className="author-guide-content">
```

Comment:
className should follow the BEM convention we use in the other front end components
`author-guide__content`